plugins/workspace-minimap/test/index.ts

```diff
@@ -12,22 +12,23 @@ import * as Blockly from 'blockly';
 import {toolboxCategories, createPlayground} from '@blockly/dev-tools';
 import {Minimap, PositionedMinimap} from '../src/index';
 
-/**
- * Create a workspace.
- * @param blocklyDiv The blockly container div.
- * @param options The Blockly options.
- * @returns The created workspace.
- */
-function createWorkspace(blocklyDiv: HTMLElement,
-    options: Blockly.BlocklyOptions): Blockly.WorkspaceSvg {
-  const workspace = Blockly.inject(blocklyDiv, options);
-
-  const minimap = new Minimap(workspace);
-  minimap.init();
-
-  return workspace;
-}
-
+// Creates the primary workspace and adds the minimap.
 const ws = Blockly.inject('primaryDiv', {toolbox: toolboxCategories});
 const mp = new Minimap(ws);
 mp.init();
+
+// Creates 100 random blocks
+for (let i = 0; i < 100; i++) {
+  const prototype = 'controls_if';
+  const block = ws.newBlock(prototype);
```

Comment:
Consider just inlining the string literal here vs stashing it in a variable first since this is the only place it's used. (Also re: the comment these are all if blocks, not random ones)